lib/PayPal/Rest/ApiContext.php

```diff
@@ -85,14 +85,20 @@ public function addRequestHeader($name, $value)
      */
     public function getRequestId()
     {
-        if ($this->requestId == null) {
-            $this->requestId = $this->generateRequestId();
-        }
-
         return $this->requestId;
     }
 
     /**
+     * Sets the request ID
+     *
+     * @param string $requestId the PayPal-Request-Id value to use
+     */
+    public function setRequestId($requestId)
+    {
+        $this->requestId = $requestId;
+    }
+
+    /**
```

Comment:
Does `generateRequestId` still have a valid use case as a way to generate a unique string without as opposed to bringing in a UUID package?